Brazilian Airports SBEG and SBRJ

RSilva79 opened the pull request:
I've added the Santos Dumon Airport (SBRJ) ( Rio de Janeiro ) and Eduardo Gomes International Airport (SBEG) a great airport in the middle of the Amazonic Forrest.
Im still developing the SBEG terrain.

Commit:
load brazilian airports

Commit:
load South American Airlines

Commit:
Created Brazilian Airports SBEG and SBRJ. Created SBEG terrain

Commit:
Created Brazilian Airports SBEG and SBRJ. Created SBEG terrain

Commit:
Created South America Airlines (Conviasa, Insel, MAP, Total Airlines) and TAP Portugal Airlines.

Commit:
Added SBRJ terrain (clone of SBGL terrain), and correctin SBEG geojson file bug.

Commit:
q quit command alias to abort, new waypoints at SBEG

Commit:
q quit command alias to abort

Commit:
waypoints corrections

Commit:
waypoints corrections

Commit:
waypoints corrections

Commit:
waypoints corrections

Commit:
waypoints corrections

Commit:
waypoints corrections

Commit:
ilha do governador

Commit:
20160124

Commit:
Ilha do governador
redução de coordenadas de 12 para 6 casas decimais

Commit:
lang = eng

tedrek commented:
There are currently conflicts and there appears to be updates to SBGL mixed in here.  Could you please fix the conflicts and let us know when you are ready with the SBEG terrain?

Commit:
language voice Vicky, added DME navmode, hold on fix (correct hold pattern).

RSilva79 committed:
RSilva79

Reverted language editing

RSilva79 committed:
RSilva79

Revert Rwy names offset

RSilva79 committed:
update sbgl.json - sid AGUA enabled

The destination 105 wasn't included at line:121 "destinations" [40, 70, 105, 205, 240, 350],

RSilva79 committed:
update sbgl.json - radio

RSilva79 committed:
update sbgl.json - radio

RSilva79 commented:
Ok, I'll fix the conflicts and make a new pull request. There are some bugs on my local repo, so I'll restart from the begining.
